examples/gymfc_nf/twins/nf1/model.sdf

```diff
@@ -739,6 +739,38 @@
           <cmdMax>0.1</cmdMax>
       </joint_control_pid>
     </plugin>
+    <!-- HOKUYO SENSOR -->
+  <gazebo reference="laser_link">
+    <sensor type="gpu_ray" name="head_hokuyo_sensor">
+      <pose>0 0 0 0 0 0</pose>
+      <visualize>false</visualize>
+      <update_rate>40</update_rate>
+      <ray>
+        <scan>
+          <horizontal>
+            <samples>1024</samples>
+            <resolution>1</resolution>
+            <min_angle>-1.57</min_angle>
+            <max_angle>1.57</max_angle>
+          </horizontal>
+        </scan>
+        <range>
+          <min>0.02</min>
+          <max>10.0</max>
+          <resolution>0.01</resolution>
+        </range>
+        <noise>
+          <type>gaussian</type>
+          <mean>0.0</mean>
+          <stddev>0.0</stddev>
+        </noise>
+      </ray>
+      <plugin name="gazebo_ros_head_hokuyo_controller" filename="libgazebo_ros_gpu_laser.so">
```

Comment:
Is ROS a required dependency for this plugin?

Reply:
Doing it this way won't work, the sensor plugin you use needs to use the interface to GymFC. If the sensor plugin doesn't publish to a topic GymFC is listening to, itll never get the values. Look here, https://github.com/wil3/gymfc#api and https://github.com/wil3/gymfc-aircraft-plugins/blob/master/src/gazebo_imu_plugin.cpp#L129 for examples.